src/components/Modals/Onboarding/OnboardingTransfer.vue

```diff
@@ -727,42 +288,26 @@ export default Vue.extend({
       }
       return courses;
     },
-    removeExam(index: number) {
-      this.displayOptions.exam.splice(index, 1);
-      if (this.countExamType(this.displayOptions.exam, 'AP') === 0) {
-        this.addExam('AP');
-      }
-      if (this.countExamType(this.displayOptions.exam, 'IB') === 0) {
-        this.addExam('IB');
-      }
-      this.getCredits();
-      this.key += 1;
+    removeExam(type: 'AP' | 'IB', index: number) {
+      const exams = type === 'AP' ? this.examsAP : this.examsIB;
+      exams.splice(index, 1);
+      if (exams.length === 0) this.addExam(type);
     },
     removeTransfer(index: number) {
-      this.displayOptions.class.splice(index, 1);
-      if (this.displayOptions.class.length === 0) {
+      this.classes.splice(index, 1);
+      if (this.classes.length === 0) {
         this.addTransfer();
       }
-      this.getCredits();
     },
     addTransfer() {
       // @ts-ignore
       this.displayOptions.class.push({ class: placeholderText, credits: 0 });
```

Comment:
Since `displayOptions` is removed from data but not from this function, I think we get an error message in the console when we click "+ add another subject" or "Add" for Transferred Course Credits and a new box does not appear as shown in the screen recording. Do you think this can be resolved in this PR since it doesn't have to do with transfer logic?